lightnode/docker/default-args.sh

```diff
@@ -0,0 +1,11 @@
+# Default arguments for building the docker image.
+# To override these values locally, create a file named `args.sh` in the same directory. 'args.sh' is ignored by git.
+
+# The location of the code to clone.
+export GIT_URL=https://github.com/Layr-Labs/eigenda.git
+
+# The name of the branch or the commit sha to clone.
+export BRANCH_OR_COMMIT=master
```

Comment:
Would it make sense to use the commit that is currently checked out? Curious about the thought process here

Reply:
I'm open to that idea, but we should weigh pros and cons.

This build process clones the repo into the docker image, as opposed to using the copy on the host file system. This adds extra overhead. The reason I took this step was that I wanted to avoid uncommitted local changes accidentally making their way into the resulting docker file, as well as requiring an explicit choice when it comes to choosing which version to build. But there is certainly additional friction to the developer in doing it this way. Let's discuss.